layouts/partials/navbar.html

```diff
@@ -21,7 +21,8 @@
         {{ if .HasChildren }}
           <li class="nav__list-item">
             <div class="optionswitch">
-              <input class="optionswitch__picker" type="checkbox" id="menuoptionpicker" hidden />
+              {{ $menuId := delimit (shuffle (seq 1 9)) "" }}
```

Comment:
What is the reason for shuffling the IDs (1 to 9) here instead of simply incrementing them?

Reply:
@lxndrblz thanks for the review! Initially I wasn't able to make the increment work, that's why I tried to use shuffle instead. But now I've finally figured this out and updated the PR to use increment for menu ID instead of shuffle.